applications/MeshingApplication/tests/cpp_tests/test_internal_interpolation_process.cpp

```diff
@@ -257,13 +257,13 @@ namespace Kratos
             Create2DModelPart(this_model_part);
 
             // Set DISTANCE and other variables
-            Vector ref_metric(3);
+            array_1d<double, 3> ref_metric;
             ref_metric[0] = 1.0;
-            ref_metric[1] = 0;
-            ref_metric[2] = 1.0;
+            ref_metric[1] = 1.0;
+            ref_metric[2] = 0;
```

Comment:
ref_metric[2] = 0.0;

Reply:
Yes, but in this case is basically aesthetic. In python can be important, because you don't declare the types, and then if you say a = 1, it will assume it is an integer. But in C we say explicitly the type, I will not produce harm.

Besides, for the json it is important too, because deduces the type from the string